pyproject.toml

```diff
@@ -42,6 +42,7 @@ test = [
 	"flake8-pyproject",
 	"mypy",
 	"black",
+	"types-shapely",
```

Comment:
shapely types aren't installed by default with the `--install-types` flag, so I install them explicitly for CI